vita3k/gui/src/app_selector.cpp

```diff
@@ -125,6 +125,7 @@ inline uint64_t current_time() {
         .count();
 }
 
+static float scroll_type, current_scroll_pos, max_scroll_pos;
```

Comment:
I wish this was attached to some kind of state.

Reply:
no is only needed here, using like other is good, no need set more variable inside state for nothing like all other Gui code